options.js

```diff
@@ -1,42 +1,65 @@
-chrome.storage.local.get(
-    ["options", "exclusionArray"],
-    ({
-        options = {
-            moveTabs: true,
-            effectWindows: false,
-            effectTabGroups: false,
-        },
-        exclusionArray = [],
-    }) => {
-        const { moveTabs, effectWindows, effectTabGroups } = options;
+
+/**
+ * Read from local storage in async instead of using callbacks
+ * Pass in a single string key or an array of keys to retrieve multiple values
+ * 
+ * @param {string|array<string>} key 
+ * @returns {object}
+ */
+const getLocalStorageKey = key => {
```

Comment:
the duplicate of these is on purpose, we can chat about how to remove that (spoiler: bundlers)